GUI.ipynb

```diff
@@ -192,9 +192,22 @@
   }
  ],
  "metadata": {
+  "kernelspec": {
```

Comment:
I have found no good way to prevent ipykernel from putting this junk in the notebook headers and polluting diffs. Perhaps someone at anaconda knows...

Reply:
I have also not found a way to do this besides removing manually, which violates the "good" constraint.

Reply:
This is what we use to clean notebooks if we have `pre-commit` installed:

https://github.com/holoviz/panel/blob/0cf1c5bcb092e43938fbe59c238beee7c3774e72/.pre-commit-config.yaml#L29-L32

Reply:
Perfect, thanks @ianthomas23 . 

For this repo, it would be reasonable to have pre-commit, and have that one as the only tool, or possibly with yaml and trim whitespace.

Reply:
Nice, thanks Ian! I can add that to this repo.